README.md

```diff
@@ -3,3 +3,12 @@
 This is a mod for the Free Libre Action RPG Engine (http://flarerpg.org) that provides the basic resources needed to launch the game to a playable state.
 
 The goal is to ease the process needed to get up and running with creating a stand-alone game on the Flare engine.
+
+##  Setup
+
+Just copy the flare-starter-pack directory from the mods/ directory to the mods/ directory for Flare. Then you can enable the mod in the Configuration menu. The flare-starter-pack is intended to be used by itself, so disable other mods like fantasycore and alpha_demo.
+
+	cd ./flare-engine
+	./flare
+
+If language configuration is not English, then change language to English.
```

Comment:
This step should no longer be necessary with [this engine change](https://github.com/clintbellanger/flare-engine/commit/5682d867bcfbdc12bda8d70b79f836c7d934d7a3), where Unifont is now bundled with the engine.